codex/erasure/erasure.nim

```diff
@@ -411,15 +426,12 @@ proc encodeData(
 
       trace "Erasure coding data", data = data[].len
 
-      try:
-        if err =? (
-          await self.asyncEncode(
-            manifest.blockSize.int, params.ecK, params.ecM, data, parity
-          )
-        ).errorOption:
-          return failure(err)
-      except CancelledError as exc:
-        raise exc
+      if err =? (
+        await self.asyncEncode(
+          manifest.blockSize.int, params.ecK, params.ecM, data, parity
+        )
+      ).errorOption:
+        return failure(err)
```

Comment:
@cnanakos This is the part I wanted you to specifically to check - I think we do not need to have to have this extra `try/except` here - especially that we do the same just a couple of lines below at a higher scope.

Same as